guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,290 @@
+
+### Ecash vs Custodial Lightning
+<div class="nodes -full-width" markdown="1">
+   <table>
+      <thead>
+         <tr>
+            <th>&nbsp;</th>
+            <th>
+               Ecash
+            </th>
+            <th>
+               Custodial Lightning
+            </th>
+         </tr>
+      </thead>
+      <tbody>
+         <tr>
+            <td>Privacy</td>
+            <td>✅</td>
+            <td>❌</td>
+         </tr>
+         <tr>
+            <td>Secure Against Theft (Rug Pulls)</td>
+            <td>❌</td>
+            <td>❌</td>
+         </tr>
+         <tr>
+            <td>IOUs</td>
+            <td>✅</td>
+            <td>✅</td>
+         </tr>
+         <tr>
+            <td>Offline Settlement</td>
+            <td>✅</td>
+            <td>❌</td>
+         </tr>
+         <tr>
+            <td>Regulatory Risk</td>
+            <td>⚠️</td>
+            <td>⚠️</td>
+         </tr>
+         <tr>
+            <td>Bearer Asset</td>
+            <td>✅</td>
+            <td>❌</td>
+         </tr>
+      </tbody>
+   </table>
+</div>
+
+* **Privacy** - Ecash uses Chaumian blinding signatures, which makes it difficult to trace transactions back to users. Custodial lightning provides less privacy as the custodian has visibility over transactions and user activities.
+
+* **Secure Against Theft (Rug Pulls)** - Operators can potentially abscond with user funds, leading to a rug pull scenario.
+
+* **IOUs** - Both systems represent IOUs from the central operator or custodian, meaning they are not direct bitcoin ownership.
+
+* **Offline Settlement** - Ecash can be transferred offline, whereas custodial Lightning transactions typically require an online connection.
+
+* **Regulatory Risk** - Both are subject to regulatory actions that can impact their operations. Mint operators could be affected by changes in laws or enforcement actions.
+
+* **Bearer Asset** - Ecash tokens are bearer assets, meaning the mint cannot freeze the token once issued.
+
+
+## Advantages of bitcoin backed ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/connected@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/connected@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/connected@1x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/connected@2x.png"
+   alt-text = "An illustration of cashu, fedimint, and lightning nodes connected in a mesh network."
+   caption = "Lightning is the connective tissue between ecash and bitcoin."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+* **Enhanced Privacy** - Ecash uses blinded signatures, ensuring that transaction details, including user identities and amounts, remain hidden. The mint does not have access to user transaction data or balances.
```

Comment:
I would maybe clarify the use of amount here. A a mint can tall the amounts involved in a individual transaction i.e. pay an invoice of x amount or swap y proofs for y proofs. What it cannot tell is the total amount of ecash anyone user has as there is no concept of accounts.

Reply:
I moved this section to the top. This now under the "Key Benefits of Bitcoin-Backed Ecash" section. Updated the copy to:

_"Blinded signatures safeguard user identities and total balances. While the mint can see amounts for individual transactions (like paying an invoice or swapping proofs), it cannot associate these with specific users or determine anyone's total ecash holdings, as there are no user accounts."_